python/tk_framework_desktopserver/certificates.py

```diff
@@ -97,12 +97,32 @@ def _check_call(self, ctx, cmd):
         :param cmd: Command to run.
 
         :raises CertificateRegistrationError: Raised when the subprocess doesn't return 0.
+
+        :returns: Command output.
         """
-        p = subprocess.Popen(cmd, stderr=subprocess.STDOUT, stdout=subprocess.PIPE, shell=True)
+        # FIXME: This should be refactored to use the Command module from this framework.
+        # However this would mean a refactor that we can't afford at the moment so we'll
+        # stick with the minimal viable fix.
+
+        # Do not use popen.check_call because it won't redirect stderr to stdout properly
+        # and it can't close stdin which causes issues in certain configurations on Windows.
+        if sys.platform == "win32":
+            # More on this Windows specific fix here: https://bugs.python.org/issue3905
+            p = subprocess.Popen(
+                cmd, shell=True,
```

Comment:
Is shell=True here necessary? If the primary use for this is certutil, it appears as though it's not necessary after a quick test. In this case it's not a huge deal because this is a private method and we can trust the input command, but in general it's best to not use it unless you're forced to.